core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -165,6 +165,9 @@ private[data] sealed abstract class KleisliInstances1 extends KleisliInstances2
 private[data] sealed abstract class KleisliInstances2 extends KleisliInstances3 {
   implicit def catsDataAlternativeForKleisli[F[_], A](implicit F0: Alternative[F]): Alternative[Kleisli[F, A, ?]] =
     new KleisliAlternative[F, A] { def F: Alternative[F] = F0 }
+
+  implicit def catsDataContravariantMonoidalForKleisli[F[_], A](implicit F0: ContravariantMonoidal[F]): ContravariantMonoidal[Kleisli[F, A, ?]] =
```

Comment:
The convention in cats is that instances that are more specific should be at higher priority. I.E. instance of `ContravariantMonoidal` should be lower in the inheritance chain than `Contravariant` and `ContravriantSemigroupal` and `Semigroupal`.

Reply:
Gotcha, will fix.